### PR TITLE
Use a consistent look for openqa tests even via SSH

### PR DESCRIPTION
**Pull request by mvidner**
https://bugzilla.suse.com/show_bug.cgi?id=920635

### CODE REVIEW

File: startup/YaST2.call [hunk 1]
```diff
@@ -341,7 +341,12 @@ function prepare_for_ssh () {
 # prepare SSH installation
 # ---
 #
-	:
+        # bsc#920635 consistent look for openqa tests even via ssh
+        if [ "$Y2FULLSCREEN" = 1 ]; then
+                log "\tFull screen mode requested"
+                Y2_UI_ARGS="--fullscreen --noborder --auto-fonts"
+        fi
+        set_inst_qt_env
```
Comment (jreidinger) [inline startup/YaST2.call]: why this change is needed? why not call it only in if statement?

Comment (jreidinger) [inline startup/YaST2.call]: respective it is missing in changes, that it backport enablement of yast theme  in ssh

Comment (mvidner) [inline startup/YaST2.call]: Right, I forgot to mention it. How about this:
```
- Use a consistent look for openqa tests even via SSH (bsc#920635)
  - Enable the installation theme.
  - Let Y2FULLSCREEN=1 enable the full screen mode.
```

Comment (jreidinger) [inline startup/YaST2.call]: yep, looks good. LGTM if you mention it
